src/models/User.ts

```
@@ -0,0 +1,36 @@
+    public profileBackground:   String  = ""
+    public joinedAt:            Date    = new Date()
+    public identity:            String  = ""
+    public department:          String  = ""
+    public grade:               String  = ""
+    public bio:                 String  = ""
+
+
+    static parseJson(json: any) : User {
+        const user = new User()
+
+        for (const [userProperty, jsonField] of Object.entries(this.DATABASE_MAP))
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)